spec.md

```diff
@@ -592,6 +594,204 @@ After implementing the first endpoint `GET /v2/catalog` documented
 [above](#catalog-management), the Service Broker will need to be registered
 with your Platform to make your services and plans available to end users.
 
+## Service Dependencies
+
+Broker authors can define dependencies of their services to other services.
+Dependencies are declared in the catalog at the plan level.
+There are three types of dependencies.
+
+* Open dependencies are arbitrary dependencies.
+  The service instance accepts any bindings and service instance IDs and
+  evaluates them at runtime.
+* Service dependencies are specified by a service or a set of plans.
+  When the service instance is provisioned, the platform has to provide a
+  binding or a service instance ID of a matching service instance.
+* Instance dependencies declare dependencies to a specific instance either
+  by a service instance ID or by a service instance alias.
+
+Dependencies are injected as parameters when a service instance is provisioned.
+The names of the parameters are declared in the catalog.
+
+The parameter values are objects and are different for bindable and
+non-bindable services. They all have the following fields in common.
+
+| Field | Type | Description |
+| --- | --- | --- |
+| service_id* | string | Service ID of the instance. |
+| service_name* | string | Service name of the instance. |
+| plan_id* | string | Plan ID of the instance. |
+| plan_name* | string | Plan name of the instance. |
+
+\* Fields with an asterisk are REQUIRED.
+
+For bindable services, the value MUST contain be the response of a binding request. 
+
+```
+{
+  "parameters": {

```

Comment:
I would recommend pulling out the dependency fields into their own field next to parameters, like: 
```
{
  "dependencies" {
    "service_id": "71ef3933-8520-4f26-b808-f483d23f94a6",
    "service_name": "mysql",
    "plan_id": "1b367d58-6f17-4cb3-9ef8-3a69e752b826",
    "plan_name": "small",
  },
  "parameters": {
    "database": {
      "credentials": {
        "uri": "mysql://mysqluser:pass@mysqlhost:3306/dbname",
        "username": "mysqluser",
        "password": "pass",
        "host": "mysqlhost",
        "port": 3306,
        "database": "dbname"
      }
    }
  }
}
```